test/integration/good/compiler-optimizations/unroll-limit.stan

```diff
@@ -0,0 +1,9 @@
+generated quantities {
+  int x;
+  for (i in 1:100) {
+    x += 1;
+  }
+  for (i in 1:20) {
```

Comment:
Mind just adding a test that tests that low/upp work, so something from 50:70 and 50 to 100?

Reply:
Also from -10 to 10 and -10 to 50 for example.

Reply:
Good idea, adding that